flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/legacy/metrics/MetricStore.java

```diff
@@ -184,6 +184,21 @@ public synchronized ComponentMetricStore getJobManagerMetricStore() {
         return ComponentMetricStore.unmodifiable(jobManager);
     }
 
+    public synchronized ComponentMetricStore getJobManagerOperatorMetricStore(
+            String jobID, String taskID) {
+        JobMetricStore job = jobID == null ? null : jobs.get(jobID);
+        if (job == null || taskID == null) {
+            return null;
+        }
+
+        TaskMetricStore task = job.getTaskMetricStore(taskID);
+        if (task == null) {
+            return null;
+        }
+
+        return ComponentMetricStore.unmodifiable(task.getJobManagerOperatorMetricStore());
+    }
```

Comment:
Feels like this could be a bit more readable
```suggestion
 public synchronized ComponentMetricStore getJobManagerOperatorMetricStore(String jobID, String taskID) {
    if (jobID == null || taskID == null) {
        return null;
    }
    JobMetricStore job = jobs.get(jobID);
    if (job == null) {
        return null;
    }
    TaskMetricStore task = job.getTaskMetricStore(taskID);
    if (task == null) {
        return null;
    }
    return ComponentMetricStore.unmodifiable(task.getJobManagerOperatorMetricStore());
}
```